README.md

```diff
@@ -130,6 +130,23 @@ bashcompinit
 source <path/to/ddev-completion.sh>
 ```
 
+#### Functional Test Code Coverage
+
+>The following dependencies are needed to run functional code coverage:
+>
+>Chromedriver v2.14
+>https://chromedriver.storage.googleapis.com/index.html?path=2.14/
+>
+>Selenium Server
+>http://www.seleniumhq.org/download/
+
+In order for coverage to be collected against functional tests, Dartium must be run against native dart code.  To launch an instance of Dartium with chromedriver you will need to configure the chromedriver instance to include the path to your dartium binary.
+```
+WebDriver driver = await createDriver(
+	desired: {'browserName': 'chrome','chromeOptions':{'binary': dartpath}});
+```
```

Comment:
you should add a note that it can be different based on how you are instantiating them (using w_webdriver_utils or just webdriver.dart)